includes/core/server/request/class-list-charge-refunds.md

```diff
@@ -0,0 +1,31 @@
+# `List_Charge_Refunds` request class
```

Comment:
Update document regarding the change in using `charge` parameter. I think you would review this doc and another doc for `Refund_Charge` too.